messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerService.java

```diff
@@ -161,16 +169,12 @@ public synchronized void start(final StartContext context) throws StartException
                     ROOT_LOGGER.aioInfo();
                 }
                 configuration.setJournalType(JournalType.NIO);
+            } else if (! AIOSequentialFileFactory.isSupported(configuration.getJournalDirectory())) {
```

Comment:
The reason is to issue a different log message if AIO loading fails and if AIO load success but AIO open fails. See https://github.com/apache/activemq-artemis/pull/425